libandroid-navigation-ui/src/main/res/layout/feedback_button_layout.xml

```diff
@@ -9,11 +9,14 @@
         android:id="@+id/feedbackFab"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/fab_feedback_margin_top"
-        android:layout_marginStart="8dp"
-        android:layout_marginEnd="16dp"
         android:layout_marginBottom="@dimen/fab_margin_bottom"
+        android:layout_marginEnd="16dp"
+        android:layout_marginLeft="8dp"
```

Comment:
Do adding a hardcoded `layout_marginLeft` and `layout_marginRight` behave well in different screen sizes and densities?

Reply:
@Guardiola31337 these were added from a lint warning - we had start / end but not left / right - both are required for the range of APIs we support within the library.  This shouldn't be adding any new margin functionality.  But, we _do_ need to revisit different margins for different screen sizes as we have previously discussed.  I think this can be done in a separate refactor.